QUEST.tsv

```diff
@@ -522,210 +522,210 @@ QUEST_20150317_000521	Seems to be asleep.
+QUEST_20150317_000635	You're my cute companion. {nl}You have scary teeth and sharp claws,{nl}but I'm not scared! {nl}You're my cute companion!
+QUEST_20150317_000636	You're my good-friend companion. {nl}The world is full of monsters, {nl}but I'm not scared. {nl}You're my good-friend companion.
 QUEST_20150317_000637	Klaipeda Goddess Statue
 QUEST_20150317_000638	Only Dievdirbys and Modestas know for whom the Goddess Statue in Klaipeda Central Plaza is for.
 QUEST_20150317_000639	Many challenged Modestas for a wager but he never lost until he died.
 QUEST_20150317_000640	Guardian of Zachariel Mausoleum
-QUEST_20150317_000641	Aside from the guardians who watch Zachariel's Mausoleum, {nl}there are other stone statues guarding inside the Mausoleum.
-QUEST_20150317_000642	Activated with flowing Mana, {nl}the guardian statues move as if they are alive and some can even talk.
-QUEST_20150317_000643	Mayvern's Opinion
-QUEST_20150317_000644	The easiest way to tell apart Clerics and Wizards is to see how they use the grace of Goddess. {nl}Clerics and Wizards can turn the grace of Goddess to different forms.
-QUEST_20150317_000645	And that power of changing grace to other forms is called Mana.
+QUEST_20150317_000641	Aside from the guardians who watch Zachariel's Mausoleum, {nl}there are stone statues guarding the Mausoleum.
+QUEST_20150317_000642	Activated with flowing mana, {nl}the guardian statues move as if they were alive. Some can even talk.
+QUEST_20150317_000643	Maven's Opinion
+QUEST_20150317_000644	The easiest way to tell Clerics and Wizards apart is to see how they use the grace of the Goddess. {nl}They can turn the grace of the Goddess to different forms.
+QUEST_20150317_000645	And that power of changing grace is called mana.
 QUEST_20150317_000646	Goddess Gabija of Fire
 QUEST_20150317_000647	Pyromancers use the blessings of Gabija, the Goddess of Fire, as their main force.
 QUEST_20150317_000648	It differs on the type of element, and depends on the grace of Goddess Gabija, {nl}but some elements may ignore oxidation.
-QUEST_20150317_000649	For this, you will need great academic standards and faith in Goddess Gabija as well.
+QUEST_20150317_000649	For this, you will need great academic standards as well as faith in Goddess Gabija.
 QUEST_20150317_000650	Will of All Things
-QUEST_20150317_000651	The simplest forms of all things are Water, Earth, Wind, and Fire, {nl}and these are controlled by the Goddess.
+QUEST_20150317_000651	The base forms of all things are Water, Earth, Wind, and Fire, {nl}and these are controlled by the Goddess.
 QUEST_20150317_000652	Elementary Fire Magic
-QUEST_20150317_000653	1. Dried environment {nl}2. Something combustible. {nl}3. Enough Mana {nl}4. The blessing of Goddess Gabija.
+QUEST_20150317_000653	1. Dried environment{nl}2. Something combustible. {nl}3. Enough mana. {nl}4. The blessing of Goddess Gabiya.
```

Comment:
I believe "Gabija" is correct here.  Referencing the wiki -> [Gabija](https://github.com/ttgmichael/EnglishTranslation/wiki/2.8.%20Names)